clients/ethereum/wrappers/interface.go

```diff
@@ -0,0 +1,30 @@
+package wrappers
+
+import (
+	"context"
+	"math/big"
+
+	"github.com/ElrondNetwork/elrond-eth-bridge/clients/ethereum/contract"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+)
+
+type genericErc20Contract interface {
+	BalanceOf(opts *bind.CallOpts, account common.Address) (*big.Int, error)
+}
+
+type multiSigContract interface {
+	GetBatch(opts *bind.CallOpts, batchNonce *big.Int) (contract.Batch, error)
+	GetRelayers(opts *bind.CallOpts) ([]common.Address, error)
+	WasBatchExecuted(opts *bind.CallOpts, batchNonce *big.Int) (bool, error)
+	ExecuteTransfer(opts *bind.TransactOpts, tokens []common.Address, recipients []common.Address, amounts []*big.Int, nonces []*big.Int, batchNonce *big.Int, signatures [][]byte) (*types.Transaction, error)
```

Comment:
maybe make a struct for ExecuteTransfer arguments

Reply:
can not do this. The struct that implement this is auto-generated